seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/sink/BaseFileSink.java

```diff
@@ -75,12 +75,6 @@ public void setJobContext(JobContext jobContext) {
         preCheckConfig();
     }
 
-    @Override
-    public void setTypeInfo(SeaTunnelRowType seaTunnelRowType) {
-        this.seaTunnelRowType = seaTunnelRowType;
-        this.fileSinkConfig = new FileSinkConfig(pluginConfig, seaTunnelRowType);
-    }
```

Comment:
Why delete this?  We still have some sink should invoke it like `OssFileSink`, `HdfsFileSink` etc.